test-cases/longevity/longevity-lwt-basic-3h.yaml

```diff
@@ -1,12 +1,14 @@
 test_duration: 180
 prepare_write_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml n=10000000 ops'(insert=1)' cl=QUORUM -mode native cql3 -rate threads=50"]
-stress_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml ops'(lwt_update_one_column=1,lwt_update_two_columns=1)' cl=QUORUM duration=160m -mode native cql3 -rate threads=30" ]
+stress_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml ops'(lwt_update_one_column=1,lwt_update_two_columns=1)' cl=QUORUM duration=160m -mode native cql3 -rate threads=30",
+             "cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml ops'(lwt_update_one_column=1,lwt_update_two_columns=1)' cl=QUORUM duration=160m -mode native cql3 -rate threads=30"
+            ]
```

Comment:
In this case I’m not sure anything will ever run on the third loader.
You should have 3 commands there.

Also it misses the purpose of using 3 loaders for the inserts because it’s not really testing the “collisions”.

Reply:
stress_cmd and stress_read_cmd run in parallel. It's 3 commands

Reply:
We told yesterday and you wanted to run prepare with round robin to prevent collisions when every loader insert into same partition and increase the time of prepare

Reply:
Roy, each command will run on the different loader. 
Yulia, I don't understand how do you prevent the insert happening on the same key?

Reply:
Prepare run just one command. In case it runs with round-robin, just one loader will run it.